docs/posts/ibisml/index.qmd

```diff
@@ -0,0 +1,897 @@
+ and perform the necessary processing for the next step.
+
+::: {.callout-tip}
+You can perform the same process with a different backend, such as
+Polars, BigQuery, Spark, or other Ibis-supported backends. Move the datasets to the
+desired database and set the Ibis backend using the reference: [Work with multiple
+backends](https://ibis-project.org/how-to/input-output/multiple-backends).
+:::
+
+### Directory structure and tables
+Since there are many data files, let's start by examining the directory structure and
+tables within the train directory:
+
+```bash
+# Change this to your directory
+tree -L 2 /Users/claypot/Downloads/home-credit-credit-risk-model-stability/parquet_files/train
+```
+
+:::{.callout-note title="Data files overview" collapse="true"}
+
+```bash
+/Users/claypot/Downloads/home-credit-credit-risk-model-stability/parquet_files/train
+├── train_applprev_1_0.parquet
+├── train_applprev_1_1.parquet
+├── train_applprev_2.parquet
+├── train_base.parquet
+├── train_credit_bureau_a_1_0.parquet
+├── train_credit_bureau_a_1_1.parquet
+├── train_credit_bureau_a_1_3.parquet
+├── train_credit_bureau_a_2_0.parquet
+├── train_credit_bureau_a_2_1.parquet
+├── train_credit_bureau_a_2_10.parquet
+├── train_credit_bureau_a_2_2.parquet
+├── train_credit_bureau_a_2_3.parquet
+├── train_credit_bureau_a_2_4.parquet
+├── train_credit_bureau_a_2_5.parquet
+├── train_credit_bureau_a_2_6.parquet
+├── train_credit_bureau_a_2_7.parquet
+├── train_credit_bureau_a_2_8.parquet
+├── train_credit_bureau_a_2_9.parquet
+├── train_credit_bureau_b_1.parquet
+├── train_credit_bureau_b_2.parquet
+├── train_debitcard_1.parquet
+├── train_deposit_1.parquet
+├── train_other_1.parquet
+├── train_person_1.parquet
+├── train_person_2.parquet
+├── train_static_0_0.parquet
+├── train_static_0_1.parquet
+├── train_static_cb_0.parquet
+├── train_tax_registry_a_1.parquet
+├── train_tax_registry_b_1.parquet
+└── train_tax_registry_c_1.parquet
+```
+
+:::
+
+The `train_base.parquet` file is the base table, while the others are feature tables.
+Let's take a quick look at these tables.
+
+#### Base table
+The base table (`train_base.parquet`) contains the unique id, a binary target flag
+and other information for the training samples. This unique ID will serve as the
+linking key for joining with other feature tables.
+
+* `case_id` - This is the unique identifier for each loan. You'll need this ID to
+  join feature tables to the base table. There are about 1.5m unique loans.
+* `date_decision` - This refers to the date when a decision was made regarding the
+  approval of the loan.
+* `WEEK_NUM` - This is the week number used for aggregation.
+* `MONTH` - This column represents the month when the approval decision was made.
+* `target` - This is the binary target flag, determined after a certain period based on
+  whether or not the client defaulted on the specific loan.
+
+#### Feature tables
+
```

Comment:
Nit: You have spaces after some of these headers and not after others; I'm surprised `prettier` doesn't complain.